src/utils/AwskmsUtils.ts

```diff
@@ -0,0 +1,100 @@
+import { GetObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import * as KMS from "@aws-sdk/client-kms";
+
+import fs from "fs";
+export interface KeyConfig {
+  keyID: string;
+  accessKeyId: string;
+  secretAccessKey: string;
+  region: string;
+  bucketName: string;
+  fileKey: string;
+}
+
+export interface AwskmsConfig {
+  [network: string]: {
+    [keyName: string]: KeyConfig;
+  };
+}
+
+interface AwsS3StorageConfig {
+  bucket: string;
+  key: string;
+}
+
+const { AWS_S3_STORAGE_CONFIG } = process.env;
+const storageConfig: AwsS3StorageConfig = AWS_S3_STORAGE_CONFIG ? JSON.parse(AWS_S3_STORAGE_CONFIG) : undefined;
+
+export function getAwskmsConfig(keys: string[]): KeyConfig[] {
+  let configOverride: AwskmsConfig = {};
+  if (process.env.AWSKMS_CONFIG) {
+    configOverride = JSON.parse(process.env.AWSKMS_CONFIG);
+  } else {
+    const overrideFname = ".AwskmsOverride.js";
+    try {
+      if (fs.existsSync(`${__dirname}/${overrideFname}`)) {
+        configOverride = require(`./${overrideFname}`);
+      }
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+    }
+  }
+
+  const keyConfigs = keys.map((keyName: string): KeyConfig => {
+    return (configOverride["mainnet"][keyName] || {}) as KeyConfig; // Hardcode to "mainnet" network. This makes no impact key retrieval.
+  });
+
+  return keyConfigs;
+}
+
+export const downloadEncryptedKey = async (config: KeyConfig): Promise<Uint8Array | undefined> => {
```

Comment:
this doesn't need to be exported? also other definitions are using `function` style, so it would be better to align